docs/dev/tutorials/cross-chain-tutorial.md

```diff
@@ -1,64 +1,115 @@
+- Use [`nvm`](https://github.com/nvm-sh/nvm) to switch to Node 14 with the command `nvm use 14`.
+:::
+
+## Project structure
 
-::: warning
+Open a terminal window, create a new folder for the project tutorial, e.g. `mkdir cross-chain-tutorial`, and `cd` into the folder.
 
-Please note that breaking changes were introduced in `zksync-web3 ^0.13.0`. The API layer now operates with `gas` and the `ergs` concept is only used internally by the VM. 
+Now create separate folders to store contracts and scripts on L1 and L2.
 
-This tutorial will be updated shortly to reflect those changes.
+```sh
+mkdir L1-governance 
+mkdir L2-counter
+```
 
+::: note
+- The `L1-governance` code is a default Hardhat project used to deploy a contract on L1.
+- The `L2-counter` code includes all zkSync dependencies and configurations for L2.
 :::
 
-## Preliminaries
+## L1 governance
 
-In this tutorial, it is assumed that you are already familiar with deploying smart contracts on zkSync. If not, please refer to the first section of the [quickstart tutorial](../building-on-zksync/hello-world.md).
+1. `cd` into `L1-governance`.
 
-It is also assumed that you already have some experience working with Ethereum.
+2. Initialize the project:
 
-## Project structure
+::: code-tabs
+@tab npm
+```sh
+npm init -y
+```
+@tab yarn
+```sh
+yarn init -y
+```
+:::
+
+3. Install hardhat:
 
-As we will deploy contracts on both L1 and L2, we'll separate this project in two different folders:
+::: code-tabs
+@tab npm
+```sh
+npm install --save-dev hardhat
+```
+@tab yarn
+```sh
+yarn add --dev hardhat
+```
+:::
 
-- `/L1-governance`: for the L1 contract, and scripts.
-- `/L2-counter`: for the L2 contract, and scripts.
+4. Run the following to set up the project:
 
-So go ahead and create these folders.
+```sh
+npx hardhat 
+```
 
-::: tip
+Select the option **Create a Typescript project** and accept the defaults for everything else.
 
-Note that the `governance` project is a default Hardhat project because it'll be used to deploy a contract just in L1, while the `counter` project includes all the zkSync dependencies and specific configuration as it'll deploy the contract in L2.
+:::info
+To interact with the zkSync bridge contract using Solidity, you need the zkSync contract interface. There are two ways to get it:
 
+- Import it from the `@matterlabs/zksync-contracts` npm package (preferred).
+- Download it from the [contracts repo](https://github.com/matter-labs/v2-testnet-contracts).
 :::
 
-## L1 governance
+5. Install the following dependencies:
 
-To initialise the project inside the `/L1-governance` folder, run `npx hardhat`, select the option "Create a Typescript project", and leave the rest of the options with the default value.
+::: code-tabs
+@tab npm
+```sh
+npm i solc@0.8.13 @typechain/hardhat @types/node ts-node typescript @nomiclabs/hardhat-etherscan @nomiclabs/hardhat-waffle ethereum-waffle ethers @openzeppelin/contracts @matterlabs/zksync-contracts
```

Comment:
We don't need to install solc with npm, I suspect we also don't need to install these other dependencies as well.

Reply:
perhaps not, i'll can check these again